dashboard/src/components/Config/AppRepoList/AppRepoList.tsx

```diff
@@ -94,27 +95,56 @@ function AppRepoList() {
   }, [cluster, kubeappsCluster, kubeappsNamespace, globalReposNamespace]);
 
   const globalRepos: IAppRepository[] = [];
-  const namespaceRepos: IAppRepository[] = [];
+  const namespacedRepos: IAppRepository[] = [];
   repos.forEach(repo => {
     repo.metadata.namespace === globalReposNamespace
       ? globalRepos.push(repo)
-      : namespaceRepos.push(repo);
+      : namespacedRepos.push(repo);
   });
 
   const tableColumns = [
     { accessor: "name", Header: "Name" },
     { accessor: "url", Header: "URL" },
+    { accessor: "plugin", Header: "Package Type" },
     { accessor: "accessLevel", Header: "Access Level" },
     { accessor: "namespace", Header: "Namespace" },
+    { accessor: "status", Header: "Status" },
     { accessor: "actions", Header: "Actions" },
   ];
   const getTableData = (targetRepos: IAppRepository[], disableControls: boolean) => {
     return targetRepos.map(repo => {
       return {
         name: getRepoNameLinkAndTooltip(cluster, repo),
         url: repo.spec?.url,
-        accessLevel: repo.spec?.auth?.header ? "Private" : "Public",
+        // TODO(agamez): the PackageRepositorySummary API doesn't expose this field. It will be added in upcoming PRs; in the meantime, set to "unknown"
+        // accessLevel: repo.type?.auth?.header ? "Private" : "Public",
+        accessLevel: "unknown",
         namespace: repo.metadata.namespace,
+        plugin: "Helm",
+        // eslint-disable-next-line no-constant-condition
+        status: true ? (
```

Comment:
Leaving this change here, but will be modified and tested in the next PRs

Reply:
Sometimes what I do in this situation, where I'm not targetting the main branch, is that I actually write the test to fail so I can't forget to fix it in the follow-up branch.